Makefile

```diff
@@ -115,7 +115,7 @@ generate-binapi: install-generator ## Generate binapi code
 .PHONY: gen-binapi-local
 gen-binapi-local: binapi-generator check-VPP_DIR ## Generate binapi code (using locally cloned VPP)
 	@make -C ${VPP_DIR} json-api-files
-	@find $(BINAPI_DIR)/*/*.ba.go -delete
+	@find $(BINAPI_DIR)/*/*.ba.go -delete || true
 	@find $(BINAPI_DIR)/* -type d -delete
```

Comment:
Good catch, probably the second line `@find $(BINAPI_DIR)/* -type d -delete` would also need the same thing ? (in case no directories exist).

Reply:
If also the second line had this, then also genuine errors (failure to delete) would be ignored. This way, the issue is fixed and genuine errors are mostly not ignored.
Explanation:
The second line: Even if a user deleted the old binapi, there will still be the file `binapi/gen.go`. Because of the way how `find` works, it will find the file and thus succeed. It is actually irrelevant if it deletes anything.
The first line: The return value is ignored on this line, but if the deletion of binapi files fails, then the deletion of directories on the second line will also fail (because of non-empty directories), so the user will still see an error message.